### ISSUE TITLE
scope: "all issues" + "disclaimer" heading

### ISSUE THREAD

**Issue by nitedog** [w3c/EasyChecks#47]
Priority: medium
Current text: "More robust evaluation is needed to evaluate all issues comprehensively"
Suggested text: "More robust evaluation is needed to make conclusive statements" *or* "More robust evaluation is needed to address more issues and make more conclusive statements"
Rationale: Even if one does not want to evaluate *all* issues, the results from these checks may not be conclusive. That is, the limitation of this resource is not only the breadth of coverage but also the depth.

**Comment by slhenry** [comment]
In context at the top:

> These checks cover just a few accessibility issues and are designed to be quick and easy, rather than definitive. A web page could seem to pass these checks, yet still have significant accessibility barriers. More robust evaluation is needed to evaluate all issues comprehensively. 

and: [context at the bottom](https://w3c.github.io/EasyChecks/#moreeval)

I think "not definitive" covers the depth point.

How about:

> More robust evaluation is needed to evaluate accessibility comprehensively.

**Comment by sharronrush** [comment]
The section title Scope: What is Covered and Not Covered is another kind of 'clunky phrase'.  I prefer something more direct such as the title suggested in the meeting:

Scope: What these checks can and can't do 

Rationale: "Not Covered" is not exactly accurate. some things are actually covered but this level of testing is not definitive.  What they *can* do is give you an idea if any accessibility considerations have been made at all. What they *can't* do is promise WCAG conformance.

**Comment by sharronrush** [comment]
...or  What these checks do (and don't do)

**Comment by vuxcaleb** [comment]
@sharronrush I agree with changing to: "Scope: What these checks can and can't do"

**Comment by vuxcaleb** [comment]
@slhenry and @nitedog proposed changes:

## Scope: What these checks can and can't do
These checks cover just a few accessibility issues and are designed to be quick and easy, rather than definitive. A web page could seem to pass these checks, yet still have accessibility barriers. More robust evaluation is needed to evaluate accessibility comprehensively. Additional evaluation guidance is available from:
- [WCAG-EM Website Accessibility Conformance Evaluation Methodology](http://www.w3.org/WAI/eval/conformance)
- [Selecting Web Accessibility Evaluation Tools](http://www.w3.org/WAI/eval/selectingtools)
- [Involving Users in Evaluating Web Accessibility](http://www.w3.org/WAI/eval/users)

**Comment by slhenry** [comment]
Note that "What is Covered" also worked as a good heading for the list of checks that is in that section.
(_/me considered "What is Included and Not Included" and "What is Addressed and Not Addressed" but I think those are even worse for the issue Sharron noted._)

"What these checks can and can't do" doesn't work because the checks don't **do** anything -- ***people*** have to do the checks. (That would work for a "tool" - e.g., "what this tool can and can't do", but not for these checks.) For this, it would need to be something more like:
> What you can do with these checks, and not do

But that seems too wordy, and not quite the key point. I think this is the point, but too wordy:
> What you can get from these checks, and what you can't

I think the paragraph says in a nice way that these checks:
* cover some issues, but not definitively
* don't cover all issues

So how do we get all that across in a **welcoming, encouraging, clear** heading?
Maybe something like
> Scope: Helpful first check; not definitive or comprehensive

**Comment by slhenry** [comment]
/me is working up another idea that uses "Scope: What these checks can and can't do"... hope to have it before the meeting...

**Comment by slhenry** [comment]
Resolved in [16 Dec 2016 telecon](https://www.w3.org/2016/12/16-eo-minutes.html#item06)
